lx_administration/storage/manager.py

```
@@ -0,0 +1,264 @@
+    expanded = value.format_map(_Env(os.environ))
+    if "{" in expanded or "}" in expanded:
+        raise ValueError(f"Unresolved placeholder in value: {expanded}")
+    return expanded
+
+
+def _default_home_dir() -> Path:
+    home_dir = str(os.getenv("HOME_DIR"))
+    if home_dir:
+        return Path(home_dir).expanduser().resolve()
+    else:
+        raise ValueError("HOME_DIR environment variable is not set.")
```

Ruff (0.14.14)

Avoid specifying long messages outside the exception class

(TRY003)